containers/unit_tests/TestBitset.hpp

```diff
@@ -253,8 +253,10 @@ void test_bitset() {
   }
 }
 
+// FIXME_HIP deadlock
+#ifndef KOKKOS_ENABLE_HIP
```

Comment:
Rebase when #2919 makes it into develop an explicitly remove the items from the list of unit test names for HIP instead of this.

Reply:
Hmm... In some of the test files, some of the tests work and some don't. Hence, I would prefer to not remove them completely.

Reply:
Sure